guides/v2.2/cloud/trouble/robots-sitemap.md

```diff
@@ -7,43 +7,47 @@ functional_areas:
 ---
 An attempt to generate and write the `sitemap.xml` file to the root directory results in the following error:
 
-```
+```terminal
 Please make sure that "/" is writable by the web-server.
 ```
 
 With {{site.data.var.ece}}, you can only write to specific directories, such as `var`, `pub/media`, `pub/static`, or `app/etc`. When you generate the `sitemap.xml` file using the Admin panel, you must specify the `/media/` path.
 
-You do not have to generate a `robots.txt` because it generates on demand and stores the contents in the database. It does not create a file, but you can view the content in your browser with the url: `<domain.your.project>/robots.txt`
+You do not have to generate a `robots.txt` because it generates on demand and stores the contents in the database. It does not create a file, but you can view the content in your browser with the `<domain.your.project>/robots.txt` file.
 
 This requires ECE-Tools version 2002.0.12 and later with an updated `.magento.app.yaml` file. See an example of these rules in the [magento-cloud repository](https://github.com/magento/magento-cloud/blob/master/.magento.app.yaml#L43-L49).
 
-#### To generate a `sitemap.xml` file in version 2.2 and later:
+#### To generate a `sitemap.xml` file in version 2.2 and later
+
+1. Access the Magento Admin panel.
+1. On the _Marketing_ menu, click **Site Map** in the _SEO & Search_ section.
+1. In the _Site Map_ view, click **Add Sitemap**.
+1. In the _New Site Map_ view, enter the following values:
 
-1.  Access the Magento Admin panel.
-1.  On the _Marketing_ menu, click **Site Map** in the _SEO & Search_ section.
-1.  In the _Site Map_ view, click **Add Sitemap**.
-1.  In the _New Site Map_ view, enter the following values:
+   -  **Filename**:`sitemap.xml`
+   -  **Path**:`/media/`
 
-    -  **Filename**:`sitemap.xml`
-    -  **Path**:`/media/`
+1. Click **Save & Generate**. The new site map becomes available in the _Site Map_ grid.
+1. Click the path in the `Link for Google` column.
```

Comment:
Why is this in backticks, but other UI areas are in italics?

Reply:
in a search for " column", i found instances where the name of a column is in `s, in bold, and with no formatting.

Reply:
Aiming for consistency within a topic, the other column names in the procedure are in italics.

Reply:
Changing  Link for Google column name to ital.